packages/bundle-size/package.json

```diff
@@ -5,12 +5,16 @@
     "report": "tsx src/report.ts",
     "pregenerate": "rm -rf src/gen",
     "generate": "buf generate buf.build/googleapis/googleapis:9475e2896f8a46d09806149f9382e538",
-    "postgenerate": "license-header ."
+    "postgenerate": "license-header .",
+    "format": "prettier --write --ignore-unknown '.' '!src/gen/**' '!.turbo'",
+    "license-header": "license-header",
+    "lint": "eslint --max-warnings 0 ."
   },
   "dependencies": {
     "@bufbuild/buf": "^1.29.0",
     "@bufbuild/protobuf": "2.0.0-beta.1",
     "@bufbuild/protoplugin": "2.0.0-beta.1",
+    "@bufbuild/protoc-gen-es": "2.0.0-beta.1",
```

Comment:
Dependencies replace make prerequisites along with the task dependencies in turbo.json.